src/main/kotlin/io/snyk/plugin/services/SnykApplicationSettingsStateService.kt

```diff
@@ -16,6 +16,9 @@ import java.time.ZoneId
 import java.util.Date
 import java.util.UUID
 
+/**
+ * If default values changed then [io.snyk.plugin.TestUtilsKt.resetSettings] MUST be changed too!
+ */
```

Comment:
good to have that reminder, would prefer the other approach (see above), though.

Reply:
I like the other approach too. But it need some research and effort to implement.
So until we implement that better/proper way, current "workaround" for resetting Settings is better then what we had before (no reset at all and flaky test due of that)